CodingChallenges/CC_092_xor/Processing/CC_092_xor/CC_092_xor.pde

```diff
@@ -0,0 +1,79 @@
+
+
+NeuralNetwork brain;
+trainingdata[] data = {
+  new trainingdata(0, 1, 1), new trainingdata(1, 0, 1), new trainingdata(0, 0, 0), new trainingdata(1, 1, 0) 
+};
+float theta;
+void setup()
+{

```

Comment:
I know this can be controversial but the style I'm using has the `{` on the same line as the function declaration / if statement / for loop / etc. I think it's good to be consistent, would you mind adopting this style?